eng/pipelines/runtime.yml

```diff
@@ -233,130 +233,148 @@ jobs:
             eq(dependencies.checkout.outputs['SetPathVars_coreclr.containsChange'], true),
             eq(variables['isFullMatrix'], true)))
 
-#
-# Build Mono debug
-# Only when libraries or mono changed
+# Build the whole product using Mono runtime
+# Only when libraries, mono or installer are changed
 #
 - template: /eng/pipelines/common/platform-matrix.yml
   parameters:
-    jobTemplate: /eng/pipelines/mono/templates/build-job.yml
+    jobTemplate: /eng/pipelines/common/global-build-job.yml
+    buildConfig: ${{ variables.debugOnPrReleaseOnRolling }}
     runtimeFlavor: mono
-    buildConfig: debug
     platforms:
-    - Android_x64
     - Android_x86
-    - Android_arm
     - Android_arm64
     - tvOS_x64
-    - tvOS_arm64
-    - iOS_x64
-    - iOS_x86
-    - iOS_arm
     - iOS_arm64
-    - OSX_x64
-    - Linux_x64
-    - Linux_arm
-    - Linux_arm64
-    - Linux_musl_x64
+    - iOS_x86
     - Browser_wasm
-    # - Linux_musl_arm64
-    - Windows_NT_x64
-    # - Windows_NT_x86
-    # - Windows_NT_arm
-    # - Windows_NT_arm64
+    - Linux_arm
     jobParameters:
+      testGroup: innerloop
+      nameSuffix: All_Mono
```

Comment:
do we need the `All_` prefix? Right now it looks like we use it everywhere, do you see other usecases where this would be different?

Reply:
Actually the prefix was not to identify the legs from being different in the yml itself, it was more for the visual perspective so that whenever people see these legs on GH or AzDo running on their PRs, they understand these legs build all the product. Maybe we can come up with a better suggestion?

Reply:
Hm yeah I just found it could be confused with meaning "all of mono".  Maybe `AllSubsets_Mono` instead to make it clear? 

Ideally this would be in front of the job name e.g. `runtime (Build AllSubsets iOS arm64 Debug Mono)` but that's probably tricky to do.

Reply:
Yeah, that'll be tricky. I'll leave it for a subsequent PR to not spin CI again 😄